### PR TITLE
MULE-10424: Define bindings to use in Mule 4 from the event

### PR DESCRIPTION
**Pull request by afelisatti**

### CODE REVIEW

File: core/src/main/java/org/mule/runtime/core/el/v2/MuleExpressionLanguage.java [hunk 1]
```diff
@@ -74,4 +87,17 @@ public ValidationResult validate(String expression) {
     return expressionExecutor.validate(expression);
   }
 
+  private void addEventBindings(MuleEvent event, BindingContext.Builder contextBuilder) {
+    Message message = event.getMessage();
+    Attributes attributes = message.getAttributes();
```
Comment (dfeist) [inline core/src/main/java/org/mule/runtime/core/el/v2/MuleExpressionLanguage.java]: Attributes should be TypedValue in Message. Ensure there is a TODO and JIRA issue.

Comment (afelisatti) [inline core/src/main/java/org/mule/runtime/core/el/v2/MuleExpressionLanguage.java]: Done: https://www.mulesoft.org/jira/browse/MULE-10774

File: core-tests/src/test/java/org/mule/runtime/core/el/MuleExpressionLanguageTestCase.java [hunk 2]
```diff
@@ -27,4 +52,37 @@ public void nullExpressionExecutor() throws Exception {
     expressionLanguage.evaluate("hey", BindingContext.builder().build());
   }
 
+  @Test
+  public void eventBindings() throws Exception {
```
Comment (dfeist) [inline core-tests/src/test/java/org/mule/runtime/core/el/MuleExpressionLanguageTestCase.java]: Can this be split out to use a test method per-binding? This method is long..

Comment (dfeist) [inline core-tests/src/test/java/org/mule/runtime/core/el/MuleExpressionLanguageTestCase.java]: Also can we use/reuse any of the existing EL tests?

**Pull request merged**